tests/Keboola/DbExtractor/RedshiftTest.php

```diff
@@ -13,11 +14,11 @@ private function runApp(Application $app)
         $expectedCsvFile = $this->dataDir .  "/in/tables/escaping.csv";
         $outputCsvFile = $this->dataDir . '/out/tables/' . $result['imported'][0]['outputTable'] . '.csv';
         $outputManifestFile = $this->dataDir . '/out/tables/' . $result['imported'][0]['outputTable'] . '.csv.manifest';
-        $manifest = Yaml::parse(file_get_contents($outputManifestFile));
+        $manifest = json_decode(file_get_contents($outputManifestFile), true);
```

Comment:
What if the configuration is in YAML? The manifest will be created in json right? Shouldn't the createManifest method from db-extractor-common be overridden with and in case the config is in yaml, create manifest in YAML?

Reply:
I think that it doesn't matter if the manifest is yaml or json, docker runner will work with it.  And since we're moving away from `yaml` (going to be completely removing yaml support soon) I think this is fine.

Reply:
OK, fine, just wasn't sure about it.